lib/puppet/application/filebucket.rb

```diff
@@ -142,6 +169,21 @@ def backup
     end
   end
 
+  def list
+    #Default fromdate : Epoch
+    fromdate = options[:fromdate] || "0:0:0 1-1-1970"
+    #Defaults to date : Today
+    todate = options[:todate]|| Time.new.strftime("%F %T")
+    begin
+      Time.parse(fromdate)
```

Comment:
Would it be possible to split the parse validation into two different calls so that the error messages can be a little better in explaining which option is formatted incorrectly?